libseccomp/tests/notify.rs

```diff
@@ -0,0 +1,124 @@
+#![cfg(libseccomp_v2_5)]
+
+use libc::{dup3, O_CLOEXEC};
+use libseccomp::notify::*;
+use libseccomp::{ScmpAction, ScmpArch, ScmpFilterContext, ScmpSyscall, Syscall};
+use std::thread;
+
+macro_rules! skip_if_not_supported {
+    () => {
+        if notify_supported().is_err() {
```

Comment:
I think you can use [`check_api`](https://docs.rs/libseccomp/0.2.3/libseccomp/fn.check_api.html) which is a public function as follows.

```rust
if !check_api(6, ScmpVersion::from((2, 5, 0))).unwrap()
```
It would be better to leave `notify_supported` as private.

Reply:
I've removed it and also removed the line from the changelog.